crates/bitcoin-da/src/service.rs

```diff
@@ -324,6 +325,12 @@ impl BitcoinService {
         // Process transaction queue.
         self.process_transaction_queue().await?;
 
+        histogram!("bitcoin_da_transaction_queue_processing_time").record(
```

Comment:
We don't seem to be defining any labels here... which means that we can use `BITCOIN_DA_METRICS`